src/main/java/net/mcreator/ui/dialogs/workspace/NewWorkspaceDialog.java

```diff
@@ -151,6 +166,18 @@ public NewWorkspaceDialog(Window w) {
 			cardLayout.show(workspacePanels, "fabric");
 		});
 
+		JToggleButton quilt = new JToggleButton(L10N.t("dialog.new_workspace.quilt.toggle"),
+				UIRES.get("16px.quilt"));

```

Comment:
At this point, we should move this code to a method returning the button, as we do with the main menu's buttons. It's a simple and easy refactoring to make the code better.

https://github.com/MCreator/MCreator/blob/d1bdaef44e8da81c8344557f519abbfe9d1a9e0f/src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java#L421

Reply:
There is no major difference between these two approaches

Reply:
I thought it would avoid redundant code and make the code smaller